rpmbuild/copr_rpmbuild/automation/fedora_review.py

```diff
@@ -39,10 +40,12 @@ def run(self):
         """
         Run `fedora-review` tool inside the `resultdir`
         """
+        branch = chroot_to_branch(self.task["chroot"])
         cmd = [
             "fedora-review", "--no-colors", "--prebuilt", "--rpm-spec",
             "--name", self.package_name,
             "--mock-config", self.mock_config_file,
+            "--define", "DISTTAG={0}".format(branch),
```

Comment:
This must be some bug in fedora-review, what am I missing?  Is the package actually having the dist tag or not? (this must be bothering normal fedora-review users, not just Copr)